api/src/backend/api/models.py

```diff
@@ -191,6 +191,7 @@ class ProviderChoices(models.TextChoices):
         AZURE = "azure", _("Azure")
         GCP = "gcp", _("GCP")
         KUBERNETES = "kubernetes", _("Kubernetes")
+        M365 = "m365", _("M365")
```

Comment:
Question here. Is `M365` used across Prowler? Or should we use `_("Microsoft 365")` for the documentation?

Reply:
M365 is used across Prowler